source/publish_datasets/processors.rst

```diff
@@ -110,6 +110,27 @@ The same coordinate anonymized several times:
 
 .. image:: processors-geomasking-en.jpg
    :alt: Geomasking
+   
+Retrieve Administrative Divisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This processor uses a **Geo Point 2D** to retrieve information (name, code and **Geo Shapes**) of administrative 
```

Comment:
The service yes, but the processor is limited (for now) to Geo Point 2D because Geo Shape lookup can become really slow. Simple shape like circle or square works great, but if you try with real world division...